lib/rrd_utils.ml

```diff
@@ -86,7 +86,7 @@ let rec setify = function
 let f_to_s f =
   match classify_float f with
   | FP_normal | FP_subnormal ->
-      Printf.sprintf "%0.4f" f
+      Printf.sprintf "%0.5g" f
```

Comment:
- the original float output in line 921 was "%.2f", so 2 fractional digits
- after this PR, it will be "%0.5g" via a consistent use of Utils.f_to_s, so up to 5 fractional digits plus removing trailing 0s
This shouldn't be a problem, as the format "%.2f" was introduced in adc51743b03c6303649d69d2b8846bd6c5a06722 as a part of a fix to generate valid json, and introducing extra fractional digits is still valid json.